starter-code/public/scripts/controllers/routes.js

```diff
@@ -1,6 +1,12 @@
 'use strict';
 var app = app || {};
 
-// TODO: Configure routes for this app with page.js, by registering each URL your app can handle, linked to a a single controller function to handle it. Note that these routes do not need to wrapped in an IIFE.
+// DONE: Configure routes for this app with page.js, by registering each URL your app can handle, linked to a a single controller function to handle it. Note that these routes do not need to wrapped in an IIFE.
+page('/', app.articleController.init );
 
-// TODO: What function do you call to activate page.js? Fire it off now, to execute. Note that it does not need to be attached to the 'app' object nor wrapped in an IIFE.
+// DONE: What function do you call to activate page.js? Fire it off now, to execute. Note that it does not need to be attached to the 'app' object nor wrapped in an IIFE.
+page('/new', app.newArticle.initNewArticlePage);
```

Comment:
We technically do not need the new or admin page because we do not have navigation for them.